app/pages/_app.js

```diff
@@ -16,12 +17,23 @@ import 'Styles/theme/simple-line-icons/css/simple-line-icons.css';
 const PageLayout = ({ children }) => children;
 
 class App extends NextApp {
+  state = {
+    hasError: false,
+    error: null,
```

Comment:
No need to save error, won't be displaying anywhere.

Reply:
Done